src/s_audio.c

```diff
@@ -82,6 +82,16 @@ int audio_isopen(void)
             || (audio_naudiooutdev > 0 && audio_audiochoutdev[0] > 0)));
 }
 
+int audio_isfixedsr(void)
+{
+#ifdef USEAPI_JACK
+    /* JACK server sets it's own samplerate */
+    return (sys_audioapi == API_JACK);
+#else
+    return 0;
```

Comment:
remove the `#else/#endif` clause and have an unconditional `return 0` at the end.

i think this makes code more readable.